File: aula7_14032025.py
numero = int(input("digite um numero:"))
for i in range(numero, -1, -1):
    print(i)



numero = int(input("digite um nuero para somar de 1 ate ele:"))
soma = 0
i = 1
while i <= numero:
    soma += i
    i += 1
    print("soma:", soma)



numero = int(input("digte um numero:"))
for i in range(1, 11):
    print(numero, "x", i, "=", numero *i)


numero = int(input("digte um numero:"))
i = 1
while i <= 10:
    print(numero, "x", i, "=", numero *i)
    i += 1   

File: aula7_14032025.js

let numero = parseInt(prompt("Digite um número:"));
for (let i = numero; i >= 0; i--) {
    console.log(i);
}



numero = parseInt(prompt("Digite um número para somar de 1 até ele:"));
let soma = 0;
let i = 1;
while (i <= numero) {
    soma += i;
    i++;
    console.log("Soma:", soma);
}



numero = parseInt(prompt("Digite um número:"));
for (let i = 1; i <= 10; i++) {
    console.log(`${numero} x ${i} = ${numero * i}`);
}



numero = parseInt(prompt("Digite um número:"));
i = 1;
while (i <= 10) {
    console.log(`${numero} x ${i} = ${numero * i}`);
    i++;
}